CHANGES.rst

```diff
@@ -6,6 +6,11 @@ documentation
 
 - Remove references to pub server [#7421]
 
+guider_cds
```

Comment:
This regtest runs the regular calwebb_image2 pipeline for FGS science-mode data and hence is not associated with the guider_cds step (which is used exclusively for FGS guide-mode data). So this entry should not have a guider_cds heading. It should be under calwebb_image2.

Reply:
Oh, and you also need a fresh version of the change log. The unrelased version is now up to 1.9.4 (1.9.3 has been released).